cg/meta/upload/scout/raredisease_config_builder.py

```diff
@@ -0,0 +1,243 @@
+import logging
+import re
+from pathlib import Path
+
+from housekeeper.store.models import Version
+
+from cg.apps.lims import LimsAPI
+from cg.apps.madeline.api import MadelineAPI
+from cg.constants.housekeeper_tags import HK_DELIVERY_REPORT_TAG
+from cg.constants.scout import (
+    GenomeBuild,
+    RAREDISEASE_CASE_TAGS,
+    RAREDISEASE_SAMPLE_TAGS,
+    UploadTrack,
+)
+from cg.constants.subject import RelationshipStatus
+from cg.meta.upload.scout.hk_tags import CaseTags, SampleTags
+from cg.meta.upload.scout.scout_config_builder import ScoutConfigBuilder
+from cg.meta.workflow.raredisease import RarediseaseAnalysisAPI
+from cg.models.scout.scout_load_config import (
+    CaseImages,
+    CustomImages,
+    Eklipse,
+    RarediseaseLoadConfig,
+    ScoutIndividual,
+    ScoutLoadConfig,
+    ScoutRarediseaseIndividual,
+)
+from cg.store.models import Analysis, Case, CaseSample
+
+LOG = logging.getLogger(__name__)
+
+
+class RarediseaseConfigBuilder(ScoutConfigBuilder):
+    def __init__(
+        self,
+        hk_version_obj: Version,
+        analysis_obj: Analysis,
+        raredisease_analysis_api: RarediseaseAnalysisAPI,
+        lims_api: LimsAPI,
+        madeline_api: MadelineAPI,
+    ):
+        super().__init__(
+            hk_version_obj=hk_version_obj,
+            analysis_obj=analysis_obj,
+            lims_api=lims_api,
+        )
+        self.case_tags: CaseTags = CaseTags(**RAREDISEASE_CASE_TAGS)
+        self.sample_tags: SampleTags = SampleTags(**RAREDISEASE_SAMPLE_TAGS)
+        self.load_config: RarediseaseLoadConfig = RarediseaseLoadConfig(
+            track=UploadTrack.RARE_DISEASE.value,
+            delivery_report=self.get_file_from_hk({HK_DELIVERY_REPORT_TAG}),
+        )
+        self.raredisease_analysis_api: RarediseaseAnalysisAPI = raredisease_analysis_api
+        self.lims_api: LimsAPI = lims_api
+        self.madeline_api: MadelineAPI = madeline_api
+
+    def build_load_config(self, rank_score_threshold: int = 5) -> None:
```

Comment:
Let's define this 5 in a constant with a meaningful name

Reply:
I will use the constant (already existing) in the PR about rank variants